test/deploy/linux/kafka/install/roles/prepare/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+- debug:
+    msg: Starting Zookeper and Kafka Server
+
+- block:
+    - name: Start Zookeeper
+      shell: cd /opt/kafka/ && sudo bin/zookeeper-server-start.sh config/zookeeper.properties &
```

Comment:
@NRhzhao It might be good to have an assertion before this statement to check the path/executable exist, and if not throw an explicit error message (that's usually helpful to us when we see a play is not working).

Reply:
@Julien4218  will update